libs/sdk-ui-kit/src/@ui/UiChip/UiChip.scss

```
@@ -7,9 +7,13 @@ $box-shadow: var(--gd-button-dropShadow, 0 -1px 0 0) var(--gd-palette-complement
 
 $active-background-color: var(--gd-palette-complementary-2);
 
-$action-border-color: var(--gd-palette-error-base-t50);
-$action-background-color: var(--gd-palette-error-lightest);
-$action-box-shadow: var(--gd-button-dropShadow, 0 -1px 0 0) var(--gd-palette-error-base-t50) inset;
+$delete-border-color: var(--gd-palette-error-base-t50);
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)